pkg/ingress/apisix_route.go

```diff
@@ -41,13 +42,19 @@ type apisixRouteController struct {
 	controller *Controller
 	workqueue  workqueue.RateLimitingInterface
 	workers    int
+
+	// svc meta key -> ApisixRoute name -> empty
```

Comment:
we should only maintain the `map` when the service is not exist? @tao12345666333

Reply:
no, the service can be modified.

Consider this one, when service be deleted and re-created, we also need the map

Reply:
OK.